cfe_internal/recommendations.cf

```diff
@@ -176,13 +176,20 @@ bundle agent postgresql_conf_recommendations
   reports:
     psql_conf_recommendations_not_kept::
       "CFEngine Recommended Settings:";
+      "You have less than $(lower)KB of total memory, please consider adding more memory."
+        if => islowerthan( "$(mem_info_data[MemTotal][1])", $(lower) );
+      "We suggest dynamically calculated values for hosts that have between $(lower) and $(upper)KB of memory.";
```

Comment:
```suggestion
```
I would leave this comment out. When do you want it to be printed? If the memory is less than lower? or between lower and upper?